rules/windows/execution_downloaded_shortcut_files.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "development"
```

Comment:
confirming not meant for prod

Reply:
yep, no prod thanks - needs ECS fields which is why `query_schema_validation = false`